kong/runloop/balancer.lua

```diff
@@ -921,7 +974,15 @@ local function init()
 
   if kong.configuration.worker_consistency == "eventual" then
     local frequency = kong.configuration.worker_state_update_frequency or 1
-    local _, err = timer_at(frequency, update_balancer_state)
+
+    local _, err
+
+    if worker_pid() == 0 then
+      _, err = timer_at(frequency, update_balancer_state_leader)
+    else
+      _, err = timer_at(frequency, update_balancer_state_follower)
+    end
+
```

Comment:
Can we just use `timer.every` instead, like we do at (it simplifies this code a lot, as you don't need to create new timers after running them):
https://github.com/Kong/kong/blob/master/kong/runloop/handler.lua#L894-L918
and
https://github.com/Kong/kong/blob/master/kong/db/strategies/postgres/connector.lua#L329